docs/pages/docs/usage/messages.mdx

```diff
@@ -157,84 +161,94 @@ t('message'); // "Hello world!"
 t('message', {name: 'Jane'}); // "Hello Jane!"
 ```
 
-### Formatting Interpolated daynamic values
+### Cardinal pluralization
 
-Values can also be formatted based on their type. You use a `{key, type, format}` argument to do that.
+To express the pluralization of a given number of items, the `plural` argument can be used:
 
-The elements of the argument are:
+```js filename="en.json"
+"message": "You have {count, plural, =0 {no followers yet} =1 {one follower} other {# followers}}."
+```
 
-- `key` is where in the input data to find the data
-- `type` is where in the input data to find the data
-- `format` is optional, and is a further refinement on how to display that type of data
+```js
+t('message', {count: 3580}); // "You have 3,580 followers."
+```
 
-The available format `type`s available are:
+Note that by using the `#` marker, the value will be [formatted as a number](/docs/usage/numbers).
 
-- [`{number}`](/docs/usage/numbers#numbers-within-messages) This type is used to format numbers in a way that is sensitive to the locale.
-- [`{date}`](/docs/usage/dates-times#dates-and-times-within-messages) This type is used to format dates in a way that is sensitive to the locale.
-- [`{time}`](/docs/usage/dates-times#dates-and-times-within-messages) This type is used to format times in a way that is sensitive to the locale.
-- [`{select}`](/docs/usage/plurals-enums#selecting-enum-based-values) This type is used to choose output by matching a value to one of many choices.
-- [`{plural}`](/docs/usage/plurals-enums#cardinal-pluralization) This type is used to choose output based on the cardinal pluralization rules of the current locale.
-- [`{selectordinal}`](/docs/usage/plurals-enums#ordinal-pluralization) This type is used to choose output based on the ordinal pluralization rules of the current locale.
+<details>
+<summary>Which plural forms are supported?</summary>
 
-<Callout type="warning">
-  **Important**: If your setup only includes the language in the locale string like this:
+Depending on the language, different forms of cardinal pluralization are supported.
 
-```js
-{
-  locales: ['en', 'ar'],
-  defaultLocale: 'en'
-}
-```
+For example, English has two forms: One for the singular (e.g. "1 _follower_") and one for everything else (e.g. "0 _followers_" and "2 _followers_"). In contrast, Chinese only has one form, while Arabic has six.
 
-Formatting might cause hydration errors because the locale strings that you have in your setup only contain the language with no extra data in it to help the [Intl API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl) match the proper locale rules, so the Intl API resorts to a `localeMatcher` algorithm, and this algorirthm can result in different formatting rules in different hosting environments.
```

Comment:
The issue that we discussed in https://github.com/amannn/next-intl/issues/528 is really related to inconsistencies with the numbering system. This might apply to other languages too, but I don't have an overview of which languages might be affected. Also the issue might get resolved in the future if Chrome implements the fix.

Because of that, I think our discussion in the issue is currently the best documentation for the issue since it discusses a practical solution for Arabic for the current version of Chrome. I don't feel confident currently making broader suggestions about other languages and it's also somewhat hard keeping this up to date.